src/instrumenter/instrument.ts

```diff
@@ -228,10 +228,11 @@ function emitAssert(
     let userAssertionHit: Statement | undefined;
 
     if (instrCtx.assertionMode === "log") {
-        const strMessage = `${annotation.id}: ${annotation.message}`;
+        const strMessage = `000000:0000:000 ${annotation.id}: ${annotation.message}`;
```

Comment:
A question out of curiosity: is there a particular reason to use padded for zeroes for initial value?

Reply:
Just as a way to remember how the source locations should be formatted. It could have just as easily be empty spaces, just made sense in my head